addons/mail/static/src/composer/composer.js

```diff
@@ -27,6 +27,14 @@ export class Composer extends Component {
         onWillUpdateProps(({ message }) => {
             this.state.value = message ? this.convertBrToLineBreak(message.body) : "";
         });
+        useEffect(
+            () => {
+                this.ref.el.style.height = "1px";
+                this.ref.el.style.height = this.ref.el.scrollHeight + "px";
+            },
+            () => [this.state.value, this.ref.el]
+        );
+        onMounted(() => this.ref.el.scrollTo({ top: 0, behavior: "instant" }));
```

Comment:
What's the purpose of this `onMounted`?

Reply:
Mimicking master behavior, i.e. having the scroll bar to the top when editing a message.

Reply:
Could you make a test for this? This LOC was not clear by itself, so coverage with a test would make it clearer we need this LOC.

Reply:
Here you go, should be fine now :+1: